app/src/main/java/com/example/homework/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.kotlin.course.calculator
+
+import com.example.kotlin.course.calculator.api.Command
+import com.example.kotlin.course.calculator.mathCommand.DivideCommand
+import com.example.kotlin.course.calculator.mathCommand.MinusCommand
+import com.example.kotlin.course.calculator.mathCommand.PlusCommand
+import com.example.kotlin.course.calculator.mathCommand.TimesCommand
+import java.math.BigDecimal
+
+class Calculator {
+    val commands = mutableMapOf<String, Command?>("*" to TimesCommand(), "/" to DivideCommand(),
+        "+" to PlusCommand(), "-" to MinusCommand())
+
+    val operators = commands.map{ it.key }
```

Comment:
при добавлении команд у тебя поле операторов не меняется, нужно не поле а свойство
```suggestion
    val operators: List<String>
        get() = commands.map { it.key }
```